extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Services} from '../../../src/services';
+import {addParamsToUrl} from '../../../src/url';
+import {dict} from '../../../src/utils/object';
+import {getData} from '../../../src/event-helper';
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {removeElement} from '../../../src/dom';
+import {userAssert} from '../../../src/log';
+
+export class AmpConnatixPsPlayer extends AMP.BaseElement {
```

Comment:
this class has a naming convention I think, AmpConnatixPlayspacePlayer or something

Reply:
We haven't known the name yet. We still need the product team response. From what I know, Ionut already asked them,